src/screens/konnectors/LauncherView.js

```diff
@@ -295,6 +303,9 @@ class LauncherView extends Component {
                 onError={this.onWorkerError}
                 injectedJavaScriptBeforeContentLoaded={run}
               />
+              {workerVisible && this.state.workerInteractionBlockerVisible && (
```

Comment:
https://www.crocoder.dev/blog/react-conditional-rendering/ 🙏

Reply:
Now use the ternary operator